xdcrDiffer to use gocbcore.v9 and gocb.v2

nelio2k opened the pull request:
Upgraded xdcrDiffer to use gocbcore.v9 and gocb.v2
Updated go mod dependencies and fixed import statments to use subdirectory imports

lileichen commented on differ/mutationDiffer.go:
```diff
@@ -552,6 +559,7 @@ func (b *batch) send() error {
 }
 
 func (b *batch) get(key string, isSource bool) {
+	//getCallbackFunc := func(result *gocbcore.GetResult, err error) {
```
This line can be removed.

nelio2k replied:
Thanks Done

nelio2k committed:
Upgraded xdcrDiffer to use gocbcore.v9 and gocb.v2
Updated go mod dependencies and fixed import statments to use subdirectory imports
